File: frontend/src/app/api-generated/models/booking-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { BookingStateEnum } from '../models/booking-state-enum';
import { UserViewModel } from '../models/user-view-model';
export interface BookingViewModel {
  bookedById?: number | null;
  bookedByUserVm?: UserViewModel;
  bookingDate?: string;
  bookingId?: number;
  bookingPlaceId?: number;
  state?: BookingStateEnum;
}

File: backend/ViewModels/BookingViewModel.cs
using BookingApp.Enums;
using BookingApp.Models;

namespace BookingApp.ViewModels
{
    public class BookingViewModel
    {
        public int BookingId { get; set; }
        public int BookingPlaceId { get; set; }
        public BookingStateEnum State { get; set; }
        public DateTime BookingDate { get; set; }  
        public int? BookedById { get; set; }
        public UserViewModel? BookedByUserVm { get; set; }
    }
}
